match/spec/runner_spec.rb

```diff
@@ -192,16 +207,16 @@
 
           Match::Runner.new.run(config)
 
-          expect(ENV[Match::Utils.environment_variable_name(app_identifier: "tools.fastlane.app",
-                                                            type: "appstore")]).to eql('736590c3-dfe8-4c25-b2eb-2404b8e65fb8')
-          expect(ENV[Match::Utils.environment_variable_name_team_id(app_identifier: "tools.fastlane.app",
-                                                                    type: "appstore")]).to eql('439BBM9367')
-          expect(ENV[Match::Utils.environment_variable_name_profile_name(app_identifier: "tools.fastlane.app",
-                                                                         type: "appstore")]).to eql('match AppStore tools.fastlane.app 1449198835')
-          profile_path = File.expand_path('~/Library/MobileDevice/Provisioning Profiles/736590c3-dfe8-4c25-b2eb-2404b8e65fb8.mobileprovision')
-          expect(ENV[Match::Utils.environment_variable_name_profile_path(app_identifier: "tools.fastlane.app",
+          expect(ENV[Match::Utils.environment_variable_name(app_identifier: "com.jigx.test.ios",
+                                                            type: "appstore")]).to eql('2746d535-7d21-403f-8718-5deee23b2366')
+          expect(ENV[Match::Utils.environment_variable_name_team_id(app_identifier: "com.jigx.test.ios",
```

Comment:
I don't think we should change these assertions unless there's a good reason (we should keep `tools.fastlane.app` instead of `com.jigx.test.ios`)

Reply:
The reason is that i am using real certs/profiles. This is because match now checks the actual contents of the files.
I can probably somehow edit it so it looks nice without destroying the checking stuff.

Reply:
I did manage to remove it from tests but the profile itself (binary file) is required to be valid - i can't remove it from there.